### ISSUE TITLE
Issue : When updating node data via the Spark Connector

### ISSUE THREAD

**Issue by ThatDaveStevens** [neo4j-contrib/neo4j-spark-connector#452]
## Actual Behavior (Mandatory)
When we use the Spark Connector with a Save Mode of Overwrite, it’s necessary to define an option “node.keys” to specify the key field to identify the node.
 
When the target field has a name containing a colon, the parsing appears to stop at the colon.
 
For example, with a CSV file like so;

`"txtBeforeColon:txtAfterColon"`
`"Eric"`
`"Fred"`
 
Calling the Spark Connector with options including;
`"node.keys", "txtBeforeColon:txtAfterColon"`
 
The error message is
~~~
java.lang.IllegalArgumentException: Write failed due to the following errors:
- Schema is missing txtBeforeColon from option `node.keys`
 ~~~

Same thing happens if we enclose the value in backticks.

## Specifications (Mandatory)

Currently used versions

### Versions

- Spark                     3.3.0
- Scala                      (n/a - we use Java 17)
- Neo4J Server     4.4.8
 
Library Dependency Versions (per Maven pom)
- spark-core_2.13                                               3.3.0
- spark-sql_2.13                                                   3.3.0
- neo4j                                                                    4.4.9
- neo4j-connector-apache-spark_2.13       4.1.5_for_spark_3
- scala-library                                                        2.13.9

## Additional information

* The code of the Spark job
* the structure of the Dataframe
* did you define the constraints/indexes?
* if you're you using any Spark Cloud provider please specify it (ie: Databricks)

**Comment by conker84** [comment]
This should be fixed in 04d41d1, can you please try by building it from source?

**Comment by conker84** [comment]
I'm closing this as it should be fixed.  Feel free to reopen
